A/ApproxFunBase/Compat.toml

```diff
@@ -1,54 +1,51 @@
 ["0-0.0.1"]
-RecipesBase = "0.5-0"
```

Comment:
Why do you need to change e.g. this? There is not RecipesBase 1? Similar on the other changes

Reply:
Ah good point. To be honest I thought `-*` and `-0` meant the same thing...

Reply:
`-0` means "up to 1"

Reply:
Ok I’ll fix it